### DISCUSSION TITLE
Why does compilescss create two files

### DISCUSSION THREAD

**Discussion by MensoMG** [jrief/django-sass-processor#165] [Q&A]
Why compilescss creates two processed * .css files, the first next to the .scss file, and the second in SASS_PROCESSOR_ROOT, with the second file everything is clear, but why does it create another one next to .scss

**Comment by jrief** [comment]
It depends on the context you're working with.

During development, you usually don't want the generated *.css file to "pollute" your assets folder. But whenever you make a deployment, you usually invoke `./manage.py collectstatic`. The the action immediately before that, shall be `./manage.py compilescss`.
